paws_collector.js

```diff
@@ -42,8 +42,9 @@ const MAX_ERROR_RETRIES = 5;
 const MAX_LOG_BATCH_SIZE = 10000;
 function getPawsParamStoreParam(){
     return new Promise((resolve, reject) => {
-        if(fs.existsSync(CREDS_FILE_PATH)){
-            return resolve(fs.readFileSync(CREDS_FILE_PATH));
+        if (fs.existsSync(CREDS_FILE_PATH) && fs.readFileSync(CREDS_FILE_PATH).length !== 0) {
+            let fileContent = process.env.ssm_direct ? fs.readFileSync(CREDS_FILE_PATH, 'utf-8') : fs.readFileSync(CREDS_FILE_PATH);
```

Comment:
should we `fs.readFileSync(CREDS_FILE_PATH, 'base64')` if `ssm_direct  == false`?

Reply:
I will update this too. I have check we are getting base64, but for be specific i will add base64.